### ISSUE TITLE
Missing glyphs?

### ISSUE THREAD

**Issue by mmindenhall** [epk/SF-Mono-Nerd-Font#4]
I was looking for a terraform glyph, which I found in the [Nerd Fonts cheat sheet](https://www.nerdfonts.com/cheat-sheet).  However, when I open the SF Mono Nerd Font in the Font Book app on my Mac, the glyph is missing.

It's not clear whether this missing glyph is due to:
- icon/glyph set intentionally omitted when you ran the patcher
- glyph (or entire icon/glyph set containing it) was added since you last ran the patcher

Is this something you can fix by re-patching?  If not, would you be willing to share the command you used when running the patcher?

Many thanks!

**Comment by mahyarmirrashed** [comment]
I believe that this is. I am also missing some glyphs like Vue.

**Comment by iofusion** [comment]
To update with missing glyphs, I downloaded the font from Apple and ran this command:

`docker run --rm -v /pathToFontsDownloadedFromApple/SF-Mono:/in -v /pathToPatchedResult/SF-Mono-NerdFont:/out nerdfonts/patcher --complete
`

guided by the patcher recommendations at https://www.nerdfonts.com

**Comment by jhildenbiddle** [comment]
Thanks, @iofusion. Worked like a charm.

**Comment by ryanburnette** [comment]
A more thorough solution.
https://ryanburnette.com/blog/sf-mono-nerdfont/

**Comment by mahyarmirrashed** [comment]
Yes, I believe I did something similar to what @ryanburnette did here to patch mine.
